commit 3af1bce6905f174fa291ddb031c2a3bbdac80e9a (refs/changes/77/4977/1)
Author: Hans Rosenfeld <hans.rosenfeld@joyent.com>
Date:   2018-10-19T16:13:08+02:00 (12 months ago)
    
    SWSUP-1214 ON-PREM - Limited Brands cannot access SSDs on R730XD

diff --git a/overlay/generic/kernel/drv/sd.conf b/overlay/generic/kernel/drv/sd.conf
index fce0d72c..43c439fd 100644
--- a/overlay/generic/kernel/drv/sd.conf
+++ b/overlay/generic/kernel/drv/sd.conf
@@ -74,6 +74,7 @@ ddi-devid-registrant=1;
 #
 sd-config-list=
 	"", "retries-timeout:1,retries-busy:1,retries-reset:1,retries-victim:2",
+	"SAMSUNG MZILS800HEHP0D3", "power-condition:false",
 	"DELL    PERC H710", "cache-nonvolatile:true",
 	"DELL    PERC H700", "cache-nonvolatile:true",
 	"DELL    PERC/6i", "cache-nonvolatile:true",
